_includes/header.html

```diff
@@ -41,10 +41,10 @@
                 </ol>
               </li>
               <li class="menu-item"><a class="link link--dia" href="{{ 'model-loader-intro' | relative_url }}">Model Loader</a></li>
-              <!-- <li class="menu-item"><a class="link link--dia" href="{{ 'blog' | relative_url }}">Blog</a></li> -->
+              <li class="menu-item"><a class="link link--dia" href="{{ 'blog' | relative_url }}">Blog</a></li>
               <li class="menu-item"><a class="link link--dia" href="{{ 'community' | relative_url }}">Community</a></li>
               <li class="menu-item">
-                <a class="link link--dia faq" href="https://github.com/webmachinelearning/">Running Code</a>
+                <a class="link link--dia faq" href="https://github.com/webmachinelearning/">The Running Code</a>
```

Comment:
I’d suggest naming this link ”GitHub”.

Better to name the link explicitly I think.